.github/workflows/Unit-Test.yml

```diff
@@ -13,55 +13,56 @@ jobs:
           GITHUB_CONTEXT: ${{ toJson(github) }}
       - name: Get all changed package files
         id: changed-ruby-files
-        uses: tj-actions/changed-files@v44
+        uses: tj-actions/changed-files@v44.5.7
         with:
           files: |
              packages/*.rb
       - name: List all changed package files
-        uses: tj-actions/changed-files@v44
+        uses: tj-actions/changed-files@v44.5.7
         if: steps.changed-ruby-files.outputs.any_changed == 'true'
+      - name: Unit Tests (x86_64)
         env:
           ALL_CHANGED_FILES: ${{ steps.changed-ruby-files.outputs.all_changed_files }}
-      - name: Unit Tests (x86_64)
         run: |
-            sudo docker run --rm -t satmandu/crewbuild:m90-x86_64 sudo -i -u chronos /bin/bash -c "
+            export PR_NUMBER="${{ github.event.pull_request.number || github.event.issue.number }}" && \
+            echo "ALL_CHANGED FILES is/are ${ALL_CHANGED_FILES}." && \
+            export CHANGED_PACKAGES="$(echo ${ALL_CHANGED_FILES} | sed -e 's,packages/,,g' -e 's,\.rb,,g' | sort)" && \
+            echo "CHANGED PACKAGES is/are ${CHANGED_PACKAGES}." && \
+            export X86_64_PACKAGES="$(for p in ${CHANGED_PACKAGES}; do grep -q "compatibility.*all\|x86_64" packages/$p.rb && echo $p; done | xargs)" && \
```

Comment:
I think we want `grep -l` instead of `grep -q` because we want to return the package names, correct?

Reply:
Oh, I see you `echo $p`.  Nevermind!